### PR TITLE
[TIMOB-24943] CLI: Install command patch for Windows platform.

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24943

**Description:**
Installing a built from sources SDK fails on Windows. 

Here I am not sure if stripping the 'unizp' command would work fine on OS X.

### CODE REVIEW

File: build/scons-install.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ function install(versionTag, next) {
 
 	// TODO Combine with unzip method in packager.js?
 	// TODO Support unzipping on windows
-	exec('/usr/bin/unzip -q -o -d "' + dest + '" "' + zipfile + '"', function (err, stdout, stderr) {
+	exec('unzip -q -o -d "' + dest + '" "' + zipfile + '"', function (err, stdout, stderr) {
```
Comment (cb1kenobi) [inline build/scons-install.js]: `unzip` is not a valid command on Windows. You should use the unzip function in `node-appc`. To do this, first require `node-appc`:

```js
var appc = require('node-appc');
```

Then unzip!

```js
appc.zip.unzip(zipfile, dest, next);
```

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

**Comment by ypbnv** [comment]
@cb1kenobi Updated.

**Review by cb1kenobi — APPROVED** [review]
Seems to work as advertised. **APPROVED**.

**Comment by ypbnv** [comment]
@jquick-axway Checked it - 'cleanbuild' on Windows does have the same problem. Since this PR introduces an issue - should I apply the changes from it for 'cleanbuild'?

**Comment by jquick-axway** [comment]
@ypbnv, I would say go for it.

And go ahead and implement the work-around I've implemented in my PR #9244. Chris has approved it over e-mail. I'm okay with closing my PR if you implement it in this one.

**Comment by ypbnv** [comment]
@jquick-axway Here is a combined PR: https://github.com/appcelerator/titanium_mobile/pull/9259

**Pull request merged**